include/powsybl/iidm/OperationalLimitsHolder.hpp

```diff
@@ -0,0 +1,80 @@
+/**
+ * Copyright (c) 2021, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+#ifndef POWSYBL_IIDM_OperationalLimitsHolder_HPP
+#define POWSYBL_IIDM_OperationalLimitsHolder_HPP
+
+#include <map>
+#include <memory>
+#include <string>
+
+#include <powsybl/iidm/LimitType.hpp>
+#include <powsybl/iidm/OperationalLimits.hpp>
+#include <powsybl/iidm/Validable.hpp>
+#include <powsybl/stdcxx/range.hpp>
+#include <powsybl/stdcxx/reference.hpp>
+
+namespace powsybl {
+
+namespace iidm {
+
+class ActivePowerLimitsAdder;
+class ApparentPowerLimitsAdder;
+class CurrentLimitsAdder;
+class Identifiable;
```

Comment:
Use forward declaration for adders here because :
`ActivePowerLimitsAdder` needs `LoadingLimitsAdder<L, A>`, which needs `OperationalLimitsHolder`, which needs `ActivePowerLimitsAdder`